Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TabbedCommandBar/TabbedCommandBarPage.xaml.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class TabbedCommandBarPage : Page
```

Comment:
At the moment, setting `IsContextual` to `True` on a tab just applies the special style to it. In the future, I want to at least add ContextualTabGroups and force all tabs with `IsContextual` to appear at the end of the list.

Reply:
Thanks @yoshiask, I was being dumb earlier and forgot I could try adding it to the live editor in the sample. 🤦‍♂️

I tried adding this snippet to the end, but the `TabbedCommandBarItem` seems to ignore it's `Visibility` property. It's probably because it's not bound to the underlying NavigationViewItem?

```xml
      xmlns:converters="using:Microsoft.Toolkit.Uwp.UI.Converters"
      mc:Ignorable="d">
      <Page.Resources>
        <converters:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter"/>
      </Page.Resources>

                <controls:TabbedCommandBarItem Header="Contextual" 
                                               IsContextual="True"
                                               Visibility="{Binding ElementName=ContextualToggle, Path=IsOn, Converter={StaticResource BoolToVisibilityConverter}}">
                </controls:TabbedCommandBarItem>
            </controls:TabbedCommandBar.MenuItems>
        </controls:TabbedCommandBar>
        <ToggleSwitch 
            x:Name="ContextualToggle"
            OffContent="Contextual Tab Off" OnContent="Contextual Tab On"/>
```

I also noticed `IsContextual` doesn't work bound either, so it may be worth just making it a regular property if we don't expect it to support being updated dynamically (which we wouldn't).

Reply:
Oh, you may not need the converter here, I thought I needed it at first, but was binding to the wrong property.

Reply:
Since the TabbedCommandBar handles the navigation/selection, how would we do this? The TabbedCommandBar has no way on knowing when the currently selected TabbedCommandBarItem becomes collapsed.

Reply:
@yoshiask I would imagine the underlying NavigationView would have some logic for this hopefully, but we'd have to test it out by making sure this example of binding to the visibility works first by actually changing the visibility of the underlying NavigationViewItem? Otherwise, in TabbedCommandBarItem itself you can [register to listen for a VisibilityChanged](https://docs.microsoft.com/uwp/api/windows.ui.xaml.dependencyobject.registerpropertychangedcallback?view=winrt-19041#Windows_UI_Xaml_DependencyObject_RegisterPropertyChangedCallback_Windows_UI_Xaml_DependencyProperty_Windows_UI_Xaml_DependencyPropertyChangedCallback_) event and do something based on that.

Reply:
I added a ToggleSwitch to the sample page, and it does work, but there's some weird initial behavior. The switch won't work until the tab bound to it is selected at least once.

https://user-images.githubusercontent.com/7549325/107172442-fdd61080-698a-11eb-95b3-e990ab1afeac.mp4